app/controllers/SwitchLanguageController.scala

```diff
@@ -23,11 +23,27 @@ import config.AppConfig
 import uk.gov.hmrc.play.language.{LanguageController, LanguageUtils}
 
 @Singleton
-class SwitchLanguageController @Inject() (appConfig: AppConfig, languageUtils: LanguageUtils, cc: MessagesControllerComponents)
+class SwitchLanguageController @Inject(
+) (appConfig: AppConfig, languageUtils: LanguageUtils, cc: MessagesControllerComponents)
     extends LanguageController(appConfig.config, languageUtils, cc) {
 
+
   override def languageMap: Map[String, Lang] = appConfig.languageMap
+  private val SwitchIndicatorKey = "switching-language"
+  private val FlashWithSwitchIndicator = Flash(Map(SwitchIndicatorKey -> "true"))
 
   // TODO requires suitable index like fallback URL
   override def fallbackURL: String = routes.AccessibilityStatementController.getPage().url
+
+  override def switchToLanguage(language: String): Action[AnyContent] = Action { implicit request =>
+    val enabled: Boolean = languageMap.get(language).exists(languageUtils.isLangAvailable)
+    val lang: Lang =
```

Comment:
IntelliJ would like some braces in if statement

Reply:
As its an override and likely to be fixed in the library, better to leave as much as found in order to see the changes. I think Intelij is wrong and the line itself would be better without the IF ;)

Reply:
Ok